### PR TITLE
Update Storage iframe [stage-3]

### PR DESCRIPTION
**Pull request by ezequielc**
@stulees I finally manage to get the widget running. Unfortunately, there was a CSS issue with storage button. I made a quick fix, but ideally we should fix the styling file. Please review. Thanks

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "trailing"     : true,  // true: Prohibit trailing whitespaces
   "maxlen"       : 200,   // {int} Max number of characters per line
   "indent"       : 2,     // {int} Number of spaces to use for indentation
-  "latedef"      : true,  // true: Require variables/functions to be defined before being used
+  "latedef"      : false,  // true: Require variables/functions to be defined before being used
```
Comment (stulees) [inline .jshintrc]: @ezequielc Can you change this value for `latedef` to "nofunc", thanks.

**Comment by stulees** [comment]
@ezequielc This repo is pointing to the latest release of `auto-scroll`, you're going to need to force it to use the previous release `1.1.0`. Regarding the styling, this repo is pointing to an old release of `widget-settings-ui-components`, hence the required fix. We can't change that release until we work on the epic for this Widget, so your fix should stay for now. Thanks

File: src/config/prod.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
-/* global CONFIG: true */
-/* exported CONFIG */
+//jshint latedef:false
```
Comment (ezequielc) [inline src/config/prod.js]: @stulees I am not sure it is an issue with current jshint version, but I could not get `CONFIG` to be validated, so I needed to add an specific `latedef:false` for the config files. Thanks!

**Comment by ezequielc** [comment]
@stulees I implemented your suggestions. Please review. Thanks!

**Comment by stulees** [comment]
@ezequielc Sorry, just realized you'll have to specify a particular release for `rv-common-i18n` to coincide with which release of `widget-settings-ui-components` is being used and the last date this widget had a commit made. Can you add the following dependency to the bower file:
```
"rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git#1.3.54"
```
Thanks

**Comment by stulees** [comment]
@ezequielc Also, can you provide a presentation to validate with, cheers.

**Comment by ezequielc** [comment]
Thanks @stulees, updated as suggested.

Here is a presentation with all 3 widgets that were updated:
http://rva.risevision.com/#/PRESENTATION_MANAGE/id=620bc22e-a2be-40a6-885b-2b0c220eef75?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

Thanks!

**Comment by stulees** [comment]
@ezequielc Can you make sure you publish your test spreadsheet to the web and also change it to be shared public. The spreadsheet widget is not displaying and I believe this is the cause.

**Comment by ezequielc** [comment]
@stulees Sorry about that, I thought I had shared it. It should work now. Cheers

**Comment by stulees** [comment]
@ezequielc Looks good :+1: , thanks!

**Comment by ezequielc** [comment]
@stulees Thanks for your help with the widgets!
@alanclayton Let me know if we are good to merge. Thanks!

**Comment by alanclayton** [comment]
@ezequielc :+1:

**Comment by ezequielc** [comment]
Thanks @alanclayton !

**Pull request merged**